monero-outreach-docs/en/quickfacts/quickfacts.md

```diff
@@ -51,42 +52,42 @@ Monero has made several large improvements since launch. Nearly all improvements
+**Monero Research Lab:** Monero is not only committed to making a fungible currency, but also to continuing research into the realm of financial privacy as it involves cryptocurrencies in general. To that end, academic researchers in fields of mathematics, physics, security, and blockchain computation have published **[research papers](https://lab.getmonero.org/)** and have many more **[goals](https://www.getmonero.org/design-goals/)** they are working toward.
 
 **Mining is accessible:** Anyone with a connected device or web browser can participate.
 
 ### REAL-WORLD IMPLICATIONS & USES
 
-Because Monero is secure, low-fee, and borderless, people can easily send money despite corrupt and broken governments or banks. This provides economic empowerment of individuals in **[oppressive countries](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** or depressed economies.
+Because Monero is secure, low-fee, and borderless, people can easily send money despite corrupt and broken governments or banks and business can be conducted without competitors snooping in critical information. This provides economic empowerment of individuals and businesses in **[oppressive countries](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)**, depressed economies, or highly competitive business environments.
 
 Private financial history protects consumers and companies from price manipulation, supply chain exploitation, economic discrimination, or the like. **[Monero is the only cryptocurrency](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** that has the features to serve as completely fungible, decentralized, electronic cash.
 
 ### TECHNICAL FUNDAMENTALS
 
-_(As of  11/14/2018)_
-+ **Amount of Active Nodes:** 1,756 (Source: https://monerohash.com/nodes-distribution.html)
-+ **Network Hash Rate:** 435.17 MH/s
-+ **Average Transactions/Hour:** 178 (30-day average)
-+ **CPU Cores Securing the Network:** 14,499,178
-+ **Monero in Circulation:** 16,570,126 XMR (Approximate)
-+ **Market Capitalization:** $1,503,928,296 USD (~0.77% of total cryptocurrency market cap)
-+ **Current Block Reward:** 3.5984 XMR
+_(As of  2/1/2019)_
++ **Amount of Active Nodes:** 1,564 (Source: https://monerohash.com/nodes-distribution.html)
++ **Network Hash Rate:** 663.2 MH/s
++ **Average Transactions/Hour:** 149 (30-day average)
++ **CPU Cores Securing the Network:** 22,234,297
++ **Monero in Circulation:** 16,758,217 XMR (Approximate)
++ **Market Capitalization:** $743,944,945 USD (~0.77% of total cryptocurrency market cap)
```

Comment:
I seems to be more 0.65% at that time

Reply:
This were data taken from the date 2/01/2019, maybe today is already at 0.65%. If we take a look at all the data today everything would change. We are actually doing it every 3 or 4 months.

Reply:
Maybe we should start on updating the file in shorter periods of time. What do you think @el00ruobuob?

Reply:
My point was that 0.77 has not been updated, and looking at February the 1st on coinmarkcap, i am closer to 0.65 than 0.77

Reply:
You are right, I took the Monero marketcap from feb 01 and the global marketcap and gave me ~0.65. I used coinmarketcap as a source. 
Marketcap
Monero - feb01 732,932,071 
https://coinmarketcap.com/currencies/monero/historical-data/?start=20181205&end=20190305
Global historical data
https://coinmarketcap.com/charts/
Global - feb01 UTC -5:00 112,421,312,048

The good thing is that we are able to verify the historical data. Wasn't sure that was something we were able to do, but well, it has been quite a learning curve. Thank you @el00ruobuob will do another commit to add this changes.

Reply:
changed, done